broker/src/main/java/io/camunda/zeebe/broker/jobstream/YieldingJobStreamErrorHandler.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.broker.jobstream;
+
+import io.camunda.zeebe.broker.Loggers;
+import io.camunda.zeebe.protocol.impl.stream.job.ActivatedJob;
+import io.camunda.zeebe.protocol.record.intent.JobIntent;
+import io.camunda.zeebe.stream.api.scheduling.TaskResultBuilder;
+
+public class YieldingJobStreamErrorHandler implements JobStreamErrorHandler {
+  @Override
+  public void handleError(
+      final ActivatedJob job, final Throwable error, final TaskResultBuilder resultBuilder) {
+    Loggers.JOB_STREAM.error("Failed to push job {}. Yielding...", job.jobKey(), error);
```

Comment:
:x: Continuing from above, once the LOG static property is declared. We can use it here:
```suggestion
    LOG.warn("Failed to push job {}. Yielding...", job.jobKey(), error);
```

:x: I think logging an ERROR is not a good idea since GCP would report it and we would always need to investigate. However, a yield is an expected operation, when a stream isn't available anymore for pushing. Maybe a WARN log would fit better?

@npepinpe @deepthidevaki do you have an opinion on this?

Reply:
@berkaycanbc we have some documentation on logging best practices in our wiki (see [here](https://github.com/camunda/zeebe/wiki/Logging#levels)). According to those docs, WARN makes the most sense here.

So you're not blocked, you can change it even now.